Source/Style.js

```diff
@@ -0,0 +1,124 @@
+// ********************************
+// # Style.js
+// # Sketch Javascript API.
+//
+// All code (C) 2016 Bohemian Coding.
+// ********************************
+
+import { WrappedObject } from './WrappedObject.js'
+
+/// A solid fill/border.
+const BCFillTypeColor    = 0
+
+/// A gradient fill/border.
+const BCFillTypeGradient = 1
+
+/// A pattern fill/border.
+const BCFillTypePattern  = 4
+
+/// A noise fill/border.
+const BCFillTypeNoise    = 5
+
+
+/**
+  Represents a Sketch layer style.
+ */
+
+export class Style extends WrappedObject {
+
+    /**
+      Make a new style object.
+
+      @param {MSStyle} style The underlying model object from Sketch.
+    */
+
+    constructor(style) {
+      if (!style) {
+        style = MSDefaultStyle.defaultStyle()
+      }
+
+      super(style)
+    }
+
+
+    /**
+      Set the borders to use for this style.
+
+      The value provided is a list of items, with each one representing a style.
+
+      Currently these values can only be strings with css-style color specifications
+      such as #ffee33 (alpha values are supported too, so #aabbccdd is valid).
+
+      These strings are used to create simple borders.
+
+      In the future the intention is to also support dictionaries allowing gradients
+      and other more complex border parameters to be specified.
+
+      @param {array} values A list of colors - each one representing a border to create.
+
+    */
+
+    set borders(value) {
+      var objects = []
+      for (var b in value) {
+        var color = MSColor.colorWithSVGString_(value[b])
```

Comment:
Leaving this here to plant a seed in your mind: we need a nice JS API to work with colours : )